moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -84,6 +84,7 @@ void moveit::core::RobotModel::buildModel(const urdf::ModelInterface &urdf_model
   root_joint_ = NULL;
   root_link_ = NULL;
   link_geometry_count_ = 0;
+  variable_count_ = 0;
```

Comment:
I don't think this belongs here - it gets set to 0 later in subfunction ``buildJointInfo()`` when it is used 

This should go in the constructor or (my fav) in the member variable declaration in C++11